src/main/java/net/mcreator/ui/validation/validators/ImageSizeMatchValidator.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.validation.validators;
+
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.validation.Validator;
+import net.mcreator.ui.workspace.resources.TextureType;
+import net.mcreator.workspace.Workspace;
+import org.gradle.internal.FileUtils;
+
+import javax.swing.*;
+
+public class ImageSizeMatchValidator implements Validator {
+
+	private String firstImage, secondImage;
+	private final TextureType type;
+	private final Workspace workspace;
+	private final boolean passIfSecondEmpty;
+
+	/**
+	 * This validator takes 2 images from a same {@link TextureType} and checks if they have the same width and height.
+	 *
+	 * @param firstImage        This is the first image of the check. This image can never be null or empty.
+	 * @param secondImage This

```

Comment:
You are missing text here as it only writes This

Reply:
Oops 😬